advice-dont-ask-to-ask.md

```diff
@@ -0,0 +1,50 @@
+# Don't ask to ask[^1]
+
+You may have been sent this document because **you asked to ask** a question,
+such as:
+
+> You: Can someone message me about <topic>?
+> You: Can anyone help me with <topic>?
+> You: Does anyone know about <topic>?
+
+This isn't particularly productive and doesn't often lead to others being able to
+help you.
+
+There are plenty of reasons why people who otherwise would help you, may not be able
+to or may not commit to replying to you. If you ask to ask, you're omitting information
+about your problem or question, since you haven't formalized it. You are instead asking
+for something more broad and a large commitment from strangers.
+
+Questions like the above cause conflict and problems because they:
+- ask others to take responsibility to take the lead
+- question people's confidence and knowledge
+- wall out people who might be able to answer your question
+- ask for someone's undivided attention in real time
+- prompt people to commit to interviewing you to get to your real questions
+- obscure any replies, which prevents it from being useful to other people
+
+Others might see this as you not having done the work to diagnose your problem - and
+conclude, why should they now put in any work to help you solve it.
+
+**Instead, take the time to form a question please.**
+
+The solution to the problem is not to ask to ask, but just to ask. Someone may see your question,
+and if they have an idea or recognize the problem, chances are they'll reply.
+
+**If you're not sure what to ask, describe your issue.**
+
+> Can someone help me with <topic>? I'm seeing <problem>, I've tried <solution>,
+> but instead of <expected>, I get <symptom>.
+
+Instead of leaving a message that only asks about a subject or topic, try describing your
+issue, even if you don't know what question to ask. If people recognize your problem or
+are interested in it, they are much more likely to help you, than if you had asked to ask.
+
+### tl;dr
```

Comment:
Same here, I'd move it to the top, "You have been sent this because ..." -> TL;DR -> "Why this is so"